underlay/src/main/java/bio/terra/tanagra/underlay/indextable/ITEntityLevelDisplayHints.java

```diff
@@ -10,13 +10,23 @@
 
 public final class ITEntityLevelDisplayHints extends IndexTable {
   public static final String TABLE_NAME = "ELDH";
-
   private final String entity;
+  private final ImmutableList<ColumnSchema> columnSchemas;
+  private final ImmutableList<String> orderBys;
 
   public ITEntityLevelDisplayHints(
       NameHelper namer, SZBigQuery.IndexData bigQueryConfig, String entity) {
     super(namer, bigQueryConfig);
     this.entity = entity;
+    this.columnSchemas =
+        ImmutableList.copyOf(
+            Arrays.stream(Column.values()).map(Column::getSchema).collect(Collectors.toList()));
+    this.orderBys =
+        ImmutableList.of(
+            Column.ATTRIBUTE_NAME.getSchema().getColumnName(),
+            Column.ENUM_VALUE.getSchema().getColumnName(),
+            Column.ENUM_DISPLAY.getSchema().getColumnName(),
```

Comment:
Best would be to push this into config but as a default I think `ENUM_DISPLAY` should be the primary sort (after attribute) for visual consistency.

Reply:
Swapped the order. will create a followup after this one to move it to a config since this is only a backend change.
reason for this PR - found this during testing a different change.

Reply:
Could be left until actually needed unless you were already going to make changes in the area. This is a good default.